### ISSUE TITLE
The Keyman's editor on Windows produce left-to-right output using Old Hungarian layout.

### ISSUE THREAD

**Issue by rovasiras** [keymanapp/keyman#8239]
The Keyman's editor on Windows produce left-to-right output using Old Hungarian layout. 
The Unicode standard describes, that Old Hungarian is right-to-left script.
Other programs work correctly.

**Comment by mcdurdin** [comment]
Yes, the Keyman Text Editor needs some effort -- not sure we'll get to this in Keyman 17 though.
